docs/Changelog.md

```diff
@@ -18587,6 +18587,94 @@ This version of the operator has been available since version 13 of the default
 </dl>
 
 ## Version 14 of the default ONNX operator set
+### <a name="BatchNormalization-14"></a>**BatchNormalization-14**</a>
+
+  Carries out batch normalization as described in the paper
+  https://arxiv.org/abs/1502.03167. Depending on the mode it is being run,
+  There is three required inputs 'X', 'mean' and 'var', in addition to one
+  optional input 'training_mode'.
+  Note that 'mean' and 'var' are expected to be the estimated statistics in
+  inference mode (training_mode=False, default),
+  and the running statistics in training mode (traning_mode=True).
+  There are multiple cases for the number of outputs, which we list below:
+  
+  Output case #1: Y, mean, var, saved_mean, saved_var (training_mode=True)
```

Comment:
Clarification: Is the output output_mean and output_var or is it saved_mean and saved_var ? Looking at the terminology below, I would have thought that the output should be output_mean and output_var. 

Suggestion: Would current_mean and current_var be more appropriate than saved_mean and saved_var ?

Reply:
+1 on more meaningful and consistent naming.

Reply:
In addition, can we rename inputs 'mean' and 'var' to 'input_mean' and 'input_var'? Agree with Hari that "current" is more understandable than "saved"

Reply:
How about using runnning_mean and running_var instead of output_mean and output_var?

Reply:
running_mean and running_var seems okay to me and I assume the other set of mean/variance will be current_mean and current_var ?